2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,11 +18,17 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+ const objectCodes = {};
+ for (element of countryCurrencyCodes){
+  objectCodes[element[0]] = element[1];
```

Comment:
try not to use words like element as it can make it harder for someone else to read through your code you could use country or currency directly. e.g. instead of element `const [country, currency]`